### PR TITLE
#37539 release notes: show number of cases without assignee in menu

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: release-notes/major9/09.26.0/valtimo-frontend-libraries.md [hunk 1]
```diff
@@ -71,4 +72,4 @@ This version has the following known issues:
 
 * **Issue2**
   * Discovered in version x.x.x
-  * Describe what can be done to work around the issue
\ No newline at end of file
+  * Describe what can be done to work around the issue
```
Comment (marijnritense) [inline release-notes/major9/09.26.0/valtimo-frontend-libraries.md:75]: ?

File: release-notes/major9/09.26.0/valtimo-frontend-libraries.md [hunk 2]
```diff
@@ -20,11 +20,12 @@ The following features were added:
   Description of the new feature goes here.
   Also link to the page that expains the feature in greater detail.
 
-* **New feature2**
+* **Number of open cases now shown in menu**
 
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+  The number of open/unassigned cases is now shown in the Valtimo menu. When the number of open cases changes, the
+  number in the menu automatically updated without reloading the page.
```
Comment (marijnritense) [inline release-notes/major9/09.26.0/valtimo-frontend-libraries.md:26]: the number in the menu *is* automatically updated without reloading the page.

Comment (luis-soares-ritense) [reply release-notes/major9/09.26.0/valtimo-frontend-libraries.md:26]: grammar nazi

Comment (Klaas-Ritense) [inline release-notes/major9/09.26.0/valtimo-frontend-libraries.md:26]:
```suggestion
  number in the menu is automatically updated without reloading the page.
```

File: release-notes/major9/09.26.0/valtimo-frontend-libraries.md (hunk 1 above)
Comment (Klaas-Ritense) [reply release-notes/major9/09.26.0/valtimo-frontend-libraries.md:75]: A newline was added to the end of the file

**Review by luis-soares-ritense — APPROVED** [review]

**Pull request merged**